AFQ/tests/test_api.py

```diff
@@ -363,24 +363,24 @@ def test_AFQ_data_waypoint():
     assert op.exists(op.join(
         myafq.data_frame['results_dir'][0],
         'bundles',
-        'sub-01_ses-01_dwi_space-RASMM_model-DTI_desc-det-AFQ-CST_L_tractography.trk'))  # noqa
+        'sub-01_ses-01_dwi_space-RASMM_model-dti_desc-det-AFQ-CST_L_tractography.trk'))  # noqa
 
     # Test creation of file with bundle indices:
     assert op.exists(op.join(
         myafq.data_frame['results_dir'][0],
-        'sub-01_ses-01_dwi_space-RASMM_model-DTI_desc-det-AFQ-clean_tractography_idx.json'))  # noqa
+        'sub-01_ses-01_dwi_space-RASMM_model-dti_desc-det-AFQ-clean_tractography_idx.json'))  # noqa
 
     tract_profiles = pd.read_csv(myafq.tract_profiles[0])
     assert tract_profiles.shape == (500, 5)
 
     myafq.plot_tract_profiles()
     assert op.exists(op.join(
         myafq.data_frame['results_dir'][0],
-        'sub-01_ses-01_dwi_space-RASMM_model-DTI_desc-det-AFQ_dti_fa_profile_plots.png'))  # noqa
+        'sub-01_ses-01_dwi_space-RASMM_model-dti_desc-det-AFQ_dti_fa_profile_plots.png'))  # noqa
```

Comment:
These file-names no longer comply with the draft of the BIDS diffusion derivatives: https://bids-specification.readthedocs.io/en/derivatives/05-derivatives/06-diffusion-derivatives.html, which has the model label for DTI with upper-case.

Reply:
I suppose I could .upper() this when its inputted, and avoid some of these problems.

Reply:
Yes. That's probably a good way to go.